mmv1/third_party/terraform/functions/zone_from_id_internal_test.go

```diff
@@ -0,0 +1,114 @@
+package functions
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/function"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestFunctionRun_zone_from_id(t *testing.T) {
+	t.Parallel()
+
+	zone := envvar.GetTestZoneFromEnv()
```

Comment:
It looks like `envvar.GetTestZoneFromEnv()` is returning an empty string and resulting in a unit test failure - I'll refactor this non-acceptance test to not depend on the environment the test is running in.

Reply:
[3ec0caa](https://github.com/GoogleCloudPlatform/magic-modules/pull/10060/commits/3ec0caa38681c8810b02c1d66ce0b6da16a28291)